app/scripts/move-popup/dimItemTag.directive.js

```diff
@@ -29,6 +29,15 @@
       $rootScope.$broadcast('dim-filter-invalidate');
       vm.item.dimInfo.save();
     };
+
+    $scope.$on('dim-item-tag', (e, args) => {
+      if (vm.item.dimInfo.tag === args.tag) {
+        delete vm.item.dimInfo.tag;
+      } else {
+        vm.item.dimInfo.tag = args.tag;
+      }
+      vm.item.dimInfo.save();
```

Comment:
may want to consider calling `$rootScope.$broadcast('dim-filter-invalidate');` here as well, if someone has an active filter and tags using a keyboard shortcut.